src/app/rundown-view/components/rundown-tooltips/piece-tooltip/piece-tooltip.component.ts

```diff
@@ -3,17 +3,21 @@ import { Tv2PieceType } from 'src/app/core/enums/tv2-piece-type'
 import { Piece } from 'src/app/core/models/piece'
 import { TooltipMousePosition } from 'src/app/core/models/tooltips'
 import { Tv2Piece } from 'src/app/core/models/tv2-piece'
-import { IconButton, IconButtonSize } from '../../../../shared/enums/icon-button'
 import { TooltipContentField } from '../../../../shared/abstractions/tooltip-content-field'
 import { Tv2PieceTooltipContentFieldService } from '../../../services/tv2-piece-tooltip-content-field.service'
 import { Media } from '../../../../shared/services/media'
+import { Icon, IconSize } from '../../../../shared/enums/icon'
 
 @Component({
   selector: 'sofie-piece-tooltip',
   templateUrl: './piece-tooltip.component.html',
   styleUrls: ['./piece-tooltip.component.scss'],
 })
 export class PieceTooltipComponent implements OnChanges {
+  protected readonly Icon = Icon
+  protected readonly IconSize = IconSize
+  protected readonly sourceUnavailableLabel: string = $localize`piece-tooltip.source-unavailable.label`
```

Comment:
I agree, this is actually something from Rasmus Tooltip PR that is coming from merging develop into my PR.
I guess since I had a merge conflict it marks it as a new change.

Reply:
It's been updated